test/DirectEditingSpec.js

```diff
@@ -340,6 +342,117 @@ describe('diagram-js-direct-editing', function() {
         });
       }));
 
+      it('should update with changed text', inject(function(canvas, directEditing, directEditingProvider) {
```

Comment:
Please separate all `it` specs with two empty lines. This is our code style to keep tests readable.